src/components/Tooltip/Tooltip.stories.mdx

```diff
@@ -35,3 +35,18 @@ An alternative use of tooltips is to provide information on a disabled actionabl
     {Template.bind({})}
   </Story>
 </Canvas>
+
+### Follow mouse
```

Comment:
Can we add some examples of a constrained tooltip as this can't be easily QA'd without it?

Reply:
I'm not quite sure what you mean sorry. You can resize your window to see this adjust, but I'm not quite sure what you mean by "constrained"...

Reply:
(Followup from convo) this only constrains by window edges, not in relation to other elements on the screen.